### ISSUE TITLE
Access to freed struct not detected

### ISSUE THREAD

**Issue by lilith218** [MPI-SWS/genmc#18]
```
#include <stdio.h>
#include <stdlib.h>

typedef struct {
	int x;
	int y;
} obj;

int main(void) {

	obj *o = malloc(sizeof(obj));
	o->x = 1;
	o->y = 2;
	free(o);
	/* this access to freed memory is not detected */
	o->y = 10;
	/* this access to freed memory is detected */
	o->x = 10;
}
```

**Comment by michaliskok** [comment]
Thanks for the bug report. GenMC currently does not report such errors for accesses to struct members or array indices other than the first. I will incorporate a fix in the next release.

**Comment by michaliskok** [comment]
Fixed in v0.6.1. The tool does not currently report partial accesses to freed memory, but I will fix this in a subsequent release.
